integration/ldt_suite.go

```diff
@@ -0,0 +1,183 @@
+
+}
+
+func (suite *localDigitalTwinsSuite) TearDownLdtSuite() {
+	suite.removeTestThing()
+	suite.TearDown()
+
+}
+
+func (suite *localDigitalTwinsSuite) getFeatureDesiredPropertyValue(featureURL string, property string) ([]byte, error) {
+	url := fmt.Sprintf(featureDesiredPropertyURLTemplate, featureURL, property)
+	return util.SendDigitalTwinRequest(suite.Cfg, http.MethodGet, url, nil)
+}
+
+func (suite *localDigitalTwinsSuite) getAllPropertiesOfFeature(featureID string) ([]byte, error) {
+	featureURL := util.GetFeatureURL(suite.thingURL, featureID)
+	url := fmt.Sprintf(featurePropertyURLTemplate, featureURL, "")
+	return util.SendDigitalTwinRequest(suite.Cfg, http.MethodGet, url, nil)
+}
+
+func (suite *localDigitalTwinsSuite) getAllDesiredPropertiesOfFeature(featureID string) ([]byte, error) {
+	featureURL := util.GetFeatureURL(suite.thingURL, featureID)
+	url := fmt.Sprintf(featureDesiredPropertyURLTemplate, featureURL, "")
+	return util.SendDigitalTwinRequest(suite.Cfg, http.MethodGet, url, nil)
+}
+
+func (suite *localDigitalTwinsSuite) getDesiredPropertyOfFeature(featureID string, property string) ([]byte, error) {
+	featureURL := util.GetFeatureURL(suite.thingURL, featureID)
+	url := fmt.Sprintf(featureDesiredPropertyURLTemplate, featureURL, property)
+	return util.SendDigitalTwinRequest(suite.Cfg, http.MethodGet, url, nil)
+}
+
+func (suite *localDigitalTwinsSuite) getPropertyOfFeature(featureID string, property string) ([]byte, error) {
+	featureURL := util.GetFeatureURL(suite.thingURL, featureID)
+	url := fmt.Sprintf(featurePropertyURLTemplate, featureURL, property)
+	return util.SendDigitalTwinRequest(suite.Cfg, http.MethodGet, url, nil)
+}
+
+func (suite *localDigitalTwinsSuite) getFeature(featureID string) ([]byte, error) {
+	featureURL := util.GetFeatureURL(suite.thingURL, featureID)
+	return util.SendDigitalTwinRequest(suite.Cfg, http.MethodGet, featureURL, nil)
+}
+
+func (suite *localDigitalTwinsSuite) getAllFeatures() ([]byte, error) {
+	URL := fmt.Sprintf("%s/features", suite.thingURL)
+	return util.SendDigitalTwinRequest(suite.Cfg, http.MethodGet, URL, nil)
+}
+
+func (suite *localDigitalTwinsSuite) getThing() ([]byte, error) {
+	return util.SendDigitalTwinRequest(suite.Cfg, http.MethodGet, suite.thingURL, nil)
+}
+
+func convertValueToJSON(value interface{}) map[string]interface{} {
+	byteValue, _ := json.Marshal(value)
+	jsonValue := make(map[string]interface{})
+	json.Unmarshal(byteValue, &jsonValue)
+
+	return jsonValue
+}
+
+func (suite *localDigitalTwinsSuite) createTestFeature(feature *model.Feature, featureID string) {
+	cmd := things.NewCommand(model.NewNamespacedIDFrom(suite.ThingCfg.DeviceID)).Twin().Feature(featureID).
+		Modify(feature)
+	msg := cmd.Envelope(protocol.WithResponseRequired(false))
+	err := suite.DittoClient.Send(msg)
+	require.NoError(suite.T(), err, "creation of test feature failed")
```

Comment:
These three lines are repeated multiple times in the functions below. Please create a separate function for them.